src/main/java/Duke.java

```diff
@@ -1,10 +1,280 @@
+                    default:
+                        continue;
+                }
+
+                if(Boolean.parseBoolean(inputSplit[1])) {
+                    task.markDone();
+                }
+                list.add(task);
+            }
+        }catch (IOException e){
+            System.out.println(e.getMessage());
+            return;
+        }
+
+        // Take user input
+        sc = new Scanner(System.in);
+        try{
+            while (sc.hasNextLine()) {
+                String input = sc.nextLine();
+                String[] input_split = input.split(" ");
+                if (input.equals("bye")) {
+                    // Stops taking inputs
+                    break;
+                } else if (input.equals("list")) {
+                    // returns list of items
+                    for (int i = 0; i < list.size(); i++) {
+                        System.out.println((1 + i) + "." + list.get(i));
+                    }
+                } else if (input_split[0].equals("done") && input_split.length == 2) {
+                    // Parse int
+                    int index = Integer.parseInt(input_split[1]) - 1;
+
+                    // Mark task as done
+                    Task task = list.get(index);
+                    task.markDone();
+
+                    //New parsed
+                    String parsedData = task.getParsedData();
+                    String newData = "";
+                    Scanner reader = new Scanner(f);
+
+                    for (int i = 0; i<list.size(); i++){
+                        if (i == index) {
+                            newData = newData + parsedData + System.lineSeparator();
+                        } else {
+                            newData = newData + reader.nextLine() + System.lineSeparator();
+                        }
+                    }
+
+
+                    saveData(dest, newData, false);
+
+                    //Notify system
+                    System.out.println("Nice! I've marked this task as done:\n" + task);
+
+                } else if (input_split[0].equals("delete")) {
+
+                    // Parse int
+                    int index = Integer.parseInt(input_split[1]) - 1;
+
+                    //Remove from queue
+                    Task removed = list.remove(index);
+
+                    //New text
+                    Scanner reader = new Scanner(f);
+                    String newData = "";
+                    for (int i = 0; i<list.size(); i++){
+                        if(i != index){
+                            newData = newData + reader.nextLine() + System.lineSeparator();
+                        }
+                    }
+
+                    saveData(dest, newData, false);
+
+                    System.out.println("Noted. I've removed this task:\n" + removed.toString() +
+                            "\nNow you have " + list.size() + " tasks in the list.");
+                } else {
+                    // Adds item to list
```

Comment:
Maybe try to keep the formatting coherent, in this case a new line after the else statement.